.github/workflows/publish-release.yml

```diff
@@ -44,7 +50,7 @@ jobs:
       - name: Merge Pull Request
         uses: juliangruber/merge-pull-request-action@9234b8714dda9a08f3d1df5b2a6a3abd7b695353 # 1.3.1
         with:
-          github-token: ${{ secrets.SVC_DEVTOOLSBOT_TOKEN }}
+          github-token: ${{ steps.app-token.outputs.token }}
```

Comment:
This will still work fine in terms of triggering GH actions runs on the created merge commit, right?

Reply:
Yeah, the app token works pretty much the same way as a user token would.